components/service/glean/docs/metrics/adding-new-metrics.md

```diff
@@ -293,6 +331,25 @@ assertTrue(Controls.refreshPressed.testHasValue())
 assertEquals(6, Controls.refreshPressed.testGetValue())
 ```
 
+## Labeled Counters
+
+Labeled counters are used to record different related counts that should sum up to a total.
+
+For example, you may want to record a count of different types of crashes for your Android application, native code crashes and uncaught exceptions:
+
+```YAML
+crash_count:
+  type: labeled_counter
+  description: >
+    Counts the number of crashes that occur in the application. ...
+  labels:
+    - uncaught_exception
+    - native_code_crash
+  ...
+```
+
+**TODO: test example here**
```

Comment:
Would it be redundant with [the other docs](https://github.com/mozilla-mobile/android-components/blob/master/components/service/glean/docs/metrics/testing-metrics.md) ?

Reply:
I think each metric type section should stand on it's own.
We do have a "what is it for?", "what do i put in metrics.yaml?", "how do i use it from my code?", and finally this section, "how do i use it in tests?".